week-3/A-array-find/exercise.js

```diff
@@ -4,6 +4,12 @@
   Using .find() (and any other array methods necessary), find the lowest pass score
 */
 
+
+const findLowestPassScore = arr => {
+  return arr.sort((a, b) => a - b).find(x => x > 60);
```

Comment:
Cool! This code seems to work, but I'm not sure you're using the syntax we've learnt in class? `findLowestPassScore ` is meant to be a function :)